pkg/k8shandler/kibana/reconciler_test.go

```diff
@@ -243,6 +254,208 @@ func TestNewKibanaPodSpecWhenProxyConfigExists(t *testing.T) {
 	checkKibanaProxyVolumesAndVolumeMounts(t, podSpec, constants.KibanaTrustedCAName)
 }
 
+func TestKibanaConsoleLinksExist(t *testing.T) {
+	_ = routev1.AddToScheme(scheme.Scheme)
+	_ = consolev1.AddToScheme(scheme.Scheme)
+	_ = loggingv1.SchemeBuilder.AddToScheme(scheme.Scheme)
+
+	cluster := &kibana.Kibana{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "kibana",
+			Namespace: constants.OpenshiftNS,
+		},
+		Spec: kibana.KibanaSpec{
+			Image:           "hub/openshift/kibana:latest",
+			ManagementState: kibana.ManagementStateManaged,
+			Replicas:        2,
+		},
+	}
+
+	caBundle := fmt.Sprint("-----BEGIN CERTIFICATE-----\n<PEM_ENCODED_CERT>\n-----END CERTIFICATE-----\n")
+	trustedCABundle := NewConfigMap(
+		constants.KibanaTrustedCAName,
+		cluster.GetNamespace(),
+		map[string]string{
+			constants.TrustedCABundleKey: caBundle,
+		},
+	)
+	trustedCABundle.ObjectMeta.Labels = make(map[string]string)
+	trustedCABundle.ObjectMeta.Labels[constants.InjectTrustedCABundleLabel] = "true"
+
+	kibanaSecret := &core.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "kibana",
+			Namespace: cluster.GetNamespace(),
+		},
+	}
+
+	kibanaProxySecret := &core.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "kibana-proxy",
+			Namespace: cluster.GetNamespace(),
+		},
+	}
+
+	proxy := &configv1.Proxy{
+		Spec: configv1.ProxySpec{
+			TrustedCA: configv1.ConfigMapNameReference{
+				Name: constants.KibanaTrustedCAName,
+			},
+		},
+	}
+
+	consoleAppLogsLink := consolev1.ConsoleLink{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      consoleAppLogsLinkName,
+			Namespace: cluster.GetNamespace(),
+		},
+		Spec: consolev1.ConsoleLinkSpec{
+			Location: consolev1.ConsoleLinkLocation("https://"),
+			ApplicationMenu: &consolev1.ApplicationMenuSpec{
+				Section: "Monitoring",
+			},
+		},
+	}
+	utils.AddOwnerRefToObject(&consoleAppLogsLink, utils.AsOwner(cluster))
```

Comment:
nit but likely dont need this in the test

Reply:
:+1: